docs/topics/feed-exports.rst

```diff
@@ -15,6 +15,15 @@ Scrapy provides this functionality out of the box with the Feed Exports, which
 allows you to generate a feed with the scraped items, using multiple
 serialization formats and storage backends.
 
+.. note::
+    Since version 1.X, Scrapy supports exporting to multiple feeds in a single crawl.
```

Comment:
Which version? Is it too late to include it in 1.7?

Reply:
Put 1.7 just in case. We can always change it later if it does not make the cut for 1.7.